dockerfiles/hpcc/templates/roxie.yaml

```diff
@@ -72,7 +75,8 @@ spec:
                 "--numChannels={{ $numChannels }}",
                 "--serverPorts={{ template "hpcc.utils.portListToCommas" $roxie.ports }}", 
                 "--topologyServers={{ $toponame }}:{{ $roxie.topoport }}",
-                "--localSlave=false"
+                "--localSlave=false", 
+                "--resolveLocally=false"
```

Comment:
Need a similar change in localroxie.yaml

Reply:
Added